Adding support for zstd, pzstd and mksquashfs using zstd/pzstd

danfrincu opened the pull request.

Commit:
Adding support for zstd, pzstd and mksquashfs using zstd/pzstd

dol-sen commented:
Hi, sorry, I've been away from home for far too long, missed a lot of mail notices.

The _x variants are deprecated, and the system uses xattr options variable.   Can you update the patch to remove the _x variant please, and I'll merge it.

Thanks 
Brian

danfrincu commented:
Hi,

No worries. I updated the PR and removed the _x variants only for what I added, wasn't sure if you meant removing the _x variant for all.

Thanks
Dan

Commit:
Removing _x variants

dol-sen commented:
yes, just for your additions.

Thank you
